### ISSUE TITLE
[9.0.100-alpha.1.24055.6] dotnet tool install command doesn't show install successful message

### ISSUE THREAD

**Issue by Junjun-zhao** [dotnet/sdk#37879]
**Application Name**: globaltools
**OS**： Windows 10 21H2
**CPU**: X64
**.NET Build Number**: dotnet-sdk-9.0.100-alpha.1.24055.6
**Also can checking it on** https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1937060

**Verify Scenarios**
1. Windows10 21h2 x64 + dotnet-sdk-8.0.101: Pass
2. Windows10 21h2 x64 + dotnet-sdk-9.0.100-alpha.1.24055.6: Fail
3. Windows 10 21h2 x64 + dotnet-sdk-8.0.200-win-x64: Fail
4. Windows 10 21h2 x64 + dotnet-sdk-8.0.102-win-x64: Pass

**Issue Description:**

When installing dotnet tools with "dotnet tool install" command, it doesn't show the installation successful message on dotnet-sdk-9.0.100-alpha.1.24055.6.

**Repro steps:**

**The machine has dotnet-sdk-9.0.100-alpha.1.24055.6 installed.**

1.Open command prompt window.
2. Input "dotnet tool install --global MarkdownSnippets.Tool" command.
4. Press "Enter"

**Expected Result:**
Tool will be installed successfully with following message shows:
![image](https://github.com/dotnet/sdk/assets/83067197/3549b472-90cd-4f2d-8f0b-7c433a7f4632)

**Actual Result:**

Tool is installed successfully but no message shows.
![image](https://github.com/dotnet/sdk/assets/83067197/f9909f60-9add-4238-8651-74d3a6cff479)

**dotnet Info:**
```
C:\Users\Appcompat>dotnet --info

.NET SDK:
 Version:           9.0.100-alpha.1.24055.6
 Commit:            ca1439c8ae
 Workload version:  9.0.100-manifests.f3b52893

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\9.0.100-alpha.1.24055.6\

.NET workloads installed:
 There are no installed workloads to display


Host:
  Version:      9.0.0-alpha.1.24054.18
  Architecture: x64
  Commit:       9325d3836d

.NET SDKs installed:
    9.0.100-alpha.1.24055.6 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 9.0.0-alpha.1.24055.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 9.0.0-alpha.1.24054.18 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 9.0.0-alpha.1.24055.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Findings:**

If set --verbosity any of m/n/d/diag, the installation message will appear. Only set --verbosity as q, it doesn't show the installation message. It looks like that "--verbosity q" set by default on dotnet-sdk-9.0.100-alpha.1.24055.6.

@dotnet-actwx-bot @dotnet/compat

**Comment by JL03-Yue** [comment]
According to recent discussions and changes, this is intentional. According to #36505, the default mode for dotnet tool install is quiet and messages are not emitted in quiet mode. This does seem to trigger some feedbacks from customer and I wonder if @baronfel has any ideas of 

(Currently, my idea is to 

- Default the tool install to minimal mode, i.e. make some changes to #36505
- In minimal mode, the `Skipping NuGet package signature verification.` message is not shown. The tool install success message shows. 

Sample
```
$ dotnet tool install Sample.tool
Tool 'XXXXXXXXX' (version '999.999.999.999') was successfully installed....
```

**Comment by Junjun-zhao** [comment]
The same error occurred on 8.0.200-rtm.24069.25 which is 8.0.200 GA build, the above verification scenarios has been updated.

**Comment by Junjun-zhao** [comment]
@JL03-Yue @Thanks for fixing this issue. This issue repro on the latest build for .NET 9 Preview 1 and .NET 8 Servicing build (8.0.200-rtm.24069.25). Could you please confirm whether it is a blocker? Will this fix be merged into the .NET 9 Preview1 and .NET 8 February update?

**Comment by Junjun-zhao** [comment]
Verified and this issue has been fixed with dotnet-sdk-9.0.100-preview.2.24113.6. Closing it
